vizro-core/tests/unit/vizro/models/_components/form/test_checklist.py

```diff
@@ -132,6 +135,19 @@ def test_create_checklist_invalid_value_format(self):
         with pytest.raises(ValidationError, match="value is not a valid list"):
             Checklist(value="A", options=["A", "B", "C"])
 
+    def test_set_action_via_validator(self, test_action_function):
```

Comment:
Maybe a more general question: does it make sense to repeat the same test for each form component as this is testing the validator and nothing component specific?

Reply:
I thought about that as well, the part that is component-specific is the `component.property` which differs per component and that is actually important to check, otherwise, no action function would work. I would also be fine with just keeping that assert line and removing the rest.

However, in the tests for the actions, we don't cover e.g. the creation of the ActionsChain yet. It's part of the next sprint, so as long as these checks are not covered elsewhere, I prefer to have them duplicated.

We can always remove later, when we cover these properly in the actions test 👍